### PR TITLE
Jetpack Section: Add Warning and Restore Status screens

### PR DESCRIPTION
**Pull request by momo-ozawa**
Part of #15191 

- Adds Warning skeleton screen
- Adds Restore Status skeleton screen

⚠️ **The following will be addressed in subsequent PRs:**
- Passing data
- API hookup
- Styling

### To test:

1. My Site > Activity Log > rewindable activity cell
2. Tap on ellipsis icon
3. Tap on `Restore`
4. Tap on `Restore to this point`
     - ✅ Should display new Warning screen
 5. Tap on `Confirm`
     - ✅ Should display new Restore Status screen

Warning | Restore Status
--- | ---
Screen Shot 2021-01-07 at 20 48 57 | Screen Shot 2021-01-07 at 20 49 00

### PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Status/RestoreStatusView.swift [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        applyStyles()
+        configure()
+    }
+
+    // MARK: - Styling
+
+    private func applyStyles() {
+        backgroundColor = .white
```
Comment (leandroalonso) [inline WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Status/RestoreStatusView.swift]: For light/dark mode:

```suggestion
        backgroundColor = .basicBackground
```

File: WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Warning/RestoreWarningView.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        applyStyles()
+        configure()
+    }
+
+    // MARK: - Styling
+
+    private func applyStyles() {
+        backgroundColor = .white
```
Comment (leandroalonso) [inline WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Warning/RestoreWarningView.swift]: For light/dark mode:

```suggestion
        backgroundColor = .basicBackground
```

**Review by leandroalonso — COMMENTED** [review]
@momo-ozawa works as described, I spotted a few issues:

* There are some hardcoded `white` colors that break the dark mode
* When I increased the font size in Simulator, I wasn't able to tap the "Restore to this point" button. Not sure if it's just a local problem tho

Also, should we change the event detail to trigger this flow? The "Rewind" button:

**Comment by momo-ozawa** [comment]
@leandroalonso 

> There are some hardcoded white colors that break the dark mode
>
> When I increased the font size in Simulator, I wasn't able to tap the "Restore to this point" button. Not sure if it's just a local problem tho

Fixed! 34bf558

> Also, should we change the event detail to trigger this flow? The "Rewind" button:

Fixed! fce71bb 

Restore Options | Warning | Restore Status
--- | --- | ---
Screen Shot 2021-01-08 at 13 48 19 | Screen Shot 2021-01-08 at 13 48 22 | Screen Shot 2021-01-08 at 13 48 25

**Review by leandroalonso — APPROVED** [review]

**Pull request merged**